runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -1376,6 +1376,15 @@ scan_udata_memory_size_helper(J9JavaVM *javaVM, char **cursor, UDATA *value, con
 			j9nls_printf(PORTLIB, J9NLS_ERROR, J9NLS_GC_OPTIONS_VALUE_OVERFLOWED, argName);
 			return false;
 		}
+#if defined(J9VM_ENV_DATA64)
+	} else if(try_scan(cursor, "T") || try_scan(cursor, "t")) {
```

Comment:
please move this up to have scan in order T, G, M, K:
```
#if defined(J9VM_ENV_DATA64)
     if(try_scan(cursor, "T") || try_scan(cursor, "t")) {
.............
     } else
#endif /* defined(J9VM_ENV_DATA64) */
    if(try_scan(cursor, "G") || try_scan(cursor, "g")) {
```

Reply:
Okay will do

Reply:
The code was copied up, but needs to be removed here.

Reply:
@dmitripivkine are there any tests for this GC code that need to be updated?

Reply:
> @dmitripivkine are there any tests for this GC code that need to be updated?

We do have implicit check in tests like https://github.com/eclipse/openj9/blob/master/test/functional/cmdLineTests/cmdLineTest_J9tests/j9tests_Java9.xml
or GC regression tests. We did have explicit parsing-only for `G`,`M`,`K` in the past but I can not find it now.
So I believe this change is good enough for now (duplicated code must be removed obviously)

Reply:
Hello, I removed the extraneous lines from `mmparse.cpp` (yet another impact of my lack of habits with squashing, but I'm learning). Now I face another strange issue: the travis CI build is finished according to the Travis [details](https://travis-ci.org/eclipse/openj9/builds/610952004?utm_source=github_status&utm_medium=notification), but current PR page fails to acknowledge that  CI build is finished for a while - even if I refresh current page in my browse.

Did I hit yet another newbie issue with PR process?

Reply:
@pshipton any idea why the PR page still displays that Travis CI is in progress while on Travis details page it's done since hours?

Reply:
Sometimes there are hiccups. It won't block merging, you don't need to be concerned. The travis build has limited value, it only tests Java 11 compilation with cmake, it doesn't run any testing.